src/main/java/dbProcs/Getter.java

```diff
@@ -1932,6 +1932,9 @@ public static String getProgress(String applicationRoot, String classId) {
         if (resultSet.getString(1) != null) {
           result +=
               "<tr><td>"
+        		  + resultAmount
```

Comment:
This seem to break the code style, if you could fix the indentation I think it should be okay and the test should pass.

You can run this check in your IDE (depends on which one you're using) to verify. Please see https://github.com/OWASP/SecurityShepherd/blob/dev/CONTRIBUTING.md